projects/Mallard/src/components/front/item/item.tsx

```diff
@@ -216,6 +258,14 @@ const superHeroImageStyles = StyleSheet.create({
     textBlock: {
         ...tappableStyles.padding,
     },
+    textStandBlock: {
+        ...tappableStyles.padding,
+        fontSize: 14,
```

Comment:
I'm a bit wary of the type sizes going all over the place around the codebase, let's talk about how to sort this out after this PR is merged tho